internal/handlers/pg/msg_count.go

```diff
@@ -61,7 +61,7 @@ func (h *Handler) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 		}
 	}
 
-	var sp pgdb.SQLParam
+	sp := new(pgdb.SQLParam)
```

Comment:
I don't understand why this change is needed.

Reply:
@AlekSi suggested using pointers for `SQLParam` in one of the functions, so I think we should move to pointers everywhere (for consistency). 

@noisersup if you have an opinion about it (where we should use values and where pointers) please share.

Reply:
So I assume that's for using a little bit less memory? :)

Reply:
I think so. To be honest, to me, in this particular case the difference is not that huge. I can agree that if all the four fields of `SQLParam` are set, we can win some memory by passing by the pointer.

Reply:
FWIW, I think we should standardize on
```go
var x T
foo(&x)
```

See https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style

Reply:
Fixed!